packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -77,6 +77,23 @@ export interface BaseServiceOptions {
    * @default - AWS Cloud Map service discovery is not enabled.
    */
   readonly cloudMapOptions?: CloudMapOptions;
+
+  /**
+   * Specifies whether to propagate the tags from the task definition or the service to the tasks in the service
+   *
+   * Valid values are: PropagateTagFromType.SERVICE or PropagateTagFromType.TASK_DEFINITION
+   *
+   * @default SERVICE
+   */
+  readonly propagateTags?: PropagateTagsFromType;
+
+  /**
+   * Specifies whether to enable Amazon ECS managed tags for the tasks within the service. For more information, see
+   * [Tagging Your Amazon ECS Resources](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-using-tags.html)
+   *
+   * @default true
+   */
+  readonly enableECSManagedTags?: boolean;
```

Comment:
If the default behavior from CloudFormation is `false`, then this is a breaking change. The default value needs to be `false` to preserve the behavior of existing, already deployed applications.

Reply:
I would consider this an additive change not a breaking change. Previously, this was false, which meant it would not be added, now it defaults to true, which wouldn't break any existing stacks.